src/Storages/Hive/HiveCommon.h

```diff
@@ -128,13 +163,15 @@ class HiveMetastoreClientFactory final : private boost::noncopyable
 
     HiveMetastoreClientPtr getOrCreate(const String & name, ContextPtr context);
 
+    static std::shared_ptr<Apache::Hadoop::Hive::ThriftHiveMetastoreClient> createThriftHiveMetastoreClient(const String & name);
+
 private:
     std::mutex mutex;
     std::map<String, HiveMetastoreClientPtr> clients;
 
-    const int conn_timeout_ms = 10000;
-    const int recv_timeout_ms = 10000;
-    const int send_timeout_ms = 10000;
+    const static int conn_timeout_ms;
```

Comment:
why do you need it here? enough to have `static constexpr int conn_timeout_ms` in cpp file.